test/functorch/test_aotdispatch.py

```diff
@@ -973,6 +973,9 @@ def assert_compiler(gm: torch.fx.GraphModule, _):
     xfail('cholesky'),
     xfail('linalg.cholesky'),
 
+    # Given input size: (s0xs1x2). Calculated output size: ...
+    skip('max_pool2d_with_indices_backward'),
```

Comment:
why is this skip and not xfail

Reply:
sorry my b https://github.com/pytorch/pytorch/pull/88979